### ISSUE TITLE
Got pure dark depth map in my own training

### ISSUE THREAD

**Issue by parallelarc** [bolianchen/pytorch_depth_from_videos_in_the_wild#16]
You know I am learning from this project but I found a strange thing. At first I generated a training dataset without 2011_09_26 in train_file.txt by accident. And therefore got pure dark depth maps in inference.  But I just added 2011_09_26 in train_file.txt and it got normal. How could this happen I wonder. Thanks for your answer.

**Comment by bolianchen** [comment]
Hi @parallelarc 
I tried removing all files prefixed with 2011_09_26 and trained a model. The results in the tensorboard seem ok.  I attached the revised train_files.txt for your reference.
![Screenshot from 2022-08-26 09-23-00](https://user-images.githubusercontent.com/39755828/186796469-fbc64974-9a2b-4dd9-83bd-c3ecaf99ec7f.png)
![Screenshot from 2022-08-26 09-23-23](https://user-images.githubusercontent.com/39755828/186796516-7cb3dd3f-cc55-417e-8bb2-b152c724c36d.png)
[train_files.txt](https://github.com/bolianchen/pytorch_depth_from_videos_in_the_wild/files/9429095/train_files.txt)

**Comment by parallelarc** [comment]
@bolianchen Thanks，I am trying to repeat the bug

**Comment by parallelarc** [comment]
Sorry to make you wait so long but that was really an arduous task for my computer. I used your provided train_files.txt but also got black depth maps (but not pure black). If I uses the full-loaded training file, it's ok. 
Btw, how can we evaluate our models? I mean,  I want to get some criterion values such like Abs Rel. I wonder if there are any of-the-shelf codes.

**Comment by parallelarc** [comment]
Hi, @bolianchen , I decide let the bug go. The training can normally progress anyway. But I still have a question, how to train with stuct2depth format.

**Comment by bolianchen** [comment]
Hi @parallelarc 
The struct2depth format concatenates 3 adjacent frames and masks into ones. It is applied in the official tensorflow repository. We believe it induces duplicates in the training data and decide to follow how monodepth2 organizes data.
